pkg/commands/setimagetag.go

```diff
@@ -69,13 +69,14 @@ func (o *setImageTagOptions) Validate(args []string) error {
 	if len(args) == 0 {
 		return errors.New("No image and newTag specified.")
 	}
+	pattern := regexp.MustCompile("^(.*):([a-zA-Z0-9._-]*)$")
```

Comment:
prefer taking this out of the method so it's only called once, i.e.

var pattern = ...

its like a const

Reply:
Updated